integration_tests/commands/async/set_data_cmd_test.go

```diff
@@ -36,115 +36,6 @@ func TestSetDataCommand(t *testing.T) {
-			delay:      []time.Duration{0, 0, 0, 0},
-		},
-		// SCARD
-		{
-			name:       "SADD & SCARD",
-			cmd:        []string{"SADD foo bar", "SADD foo baz", "SCARD foo"},
-			expected:   []interface{}{int64(1), int64(1), int64(2)},
-			assertType: []string{"equal", "equal", "equal"},
-			delay:      []time.Duration{0, 0, 0},
-		},
-		{
-			name:       "SADD & CARD with non existing key",
-			cmd:        []string{"SADD foo bar", "SADD foo baz", "SCARD bar"},
-			expected:   []interface{}{int64(1), int64(1), int64(0)},
-			assertType: []string{"equal", "equal", "equal"},
-			delay:      []time.Duration{0, 0, 0},
-		},
-		{
-			name:       "SADD & SCARD with wrong key type",
-			cmd:        []string{"SET foo bar", "SCARD foo"},
-			expected:   []interface{}{"OK", "WRONGTYPE Operation against a key holding the wrong kind of value"},
-			assertType: []string{"equal", "equal"},
-			delay:      []time.Duration{0, 0},
-		},
-		// SMEMBERS
-		{
-			name:       "SADD & SMEMBERS",
-			cmd:        []string{"SADD foo bar", "SADD foo baz", "SMEMBERS foo"},
-			expected:   []interface{}{int64(1), int64(1), []any{"bar", "baz"}},
-			assertType: []string{"equal", "equal", "equal"},
-			delay:      []time.Duration{0, 0, 0},
-		},
-		{
-			name:       "SADD & SMEMBERS with non existing key",
-			cmd:        []string{"SMEMBERS foo"},
-			expected:   []interface{}{[]any{}},
-			assertType: []string{"equal"},
-			delay:      []time.Duration{0},
-		},
-		{
-			name:       "SADD & SMEMBERS with wrong key type",
-			cmd:        []string{"SET foo bar", "SMEMBERS foo"},
-			expected:   []interface{}{"OK", "WRONGTYPE Operation against a key holding the wrong kind of value"},
-			assertType: []string{"equal", "equal"},
-			delay:      []time.Duration{0, 0},
-		},
-		// SREM
-		{
-			name:       "SADD & SREM",
-			cmd:        []string{"SADD foo bar", "SADD foo baz", "SREM foo bar", "SMEMBERS foo"},
-			expected:   []interface{}{int64(1), int64(1), int64(1), []any{"baz"}},
-			assertType: []string{"equal", "equal", "equal", "equal"},
-			delay:      []time.Duration{0, 0, 0, 0},
-		},
-		{
-			name:       "SADD & SREM with non existing key",
-			cmd:        []string{"SREM foo bar"},
-			expected:   []interface{}{int64(0)},
-			assertType: []string{"equal"},
-			delay:      []time.Duration{0},
-		},
-		{
-			name:       "SADD & SREM with wrong key type",
-			cmd:        []string{"SET foo bar", "SREM foo bar"},
-			expected:   []interface{}{"OK", "WRONGTYPE Operation against a key holding the wrong kind of value"},
-			assertType: []string{"equal", "equal"},
-			delay:      []time.Duration{0, 0},
-		},
-		{
-			name:       "SADD & SREM with non existing value",
-			cmd:        []string{"SADD foo bar baz bax", "SMEMBERS foo", "SREM foo bat", "SMEMBERS foo"},
-			expected:   []interface{}{int64(3), []any{"bar", "baz", "bax"}, int64(0), []any{"bar", "baz", "bax"}},
-			assertType: []string{"equal", "equal", "equal", "equal"},
-			delay:      []time.Duration{0, 0, 0, 0},
-		},
```

Comment:
i have left the sdiff/sinter tests in this file as those are not being migrated as a part of this pr